src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -65,6 +65,9 @@ namespace subgraph
 
 variables {V : Type u} {G : simple_graph V}
 
+lemma adj_irrefl {v : V} (G' : subgraph G) : ¬G'.adj v v :=
```

Comment:
I know Yael wants this to be `adj_irrefl`, but for consistency `loopless` would be better for now. If we decide to rename it, we can have a separate PR for that.